templates/nextflow-aurora-mysql.yaml

```diff
@@ -310,11 +349,11 @@ Resources:
           log.setLevel(logging.DEBUG)
 
           def submit_sql(sql):
+            log.debug(f'Submitting SQL: {sql}')
             return boto3.client('rds-data').execute_statement(
               secretArn=environ['MASTER_SECRET_ARN'],
               database=environ['DB_NAME'],
               resourceArn=environ['CLUSTER_ARN'],
-              schema='mysql',
```

Comment:
the boto3 rds-data client [no longer supports schema](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds-data/client/execute_statement.html).